config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-presubmits.yaml

```diff
@@ -0,0 +1,25 @@
+presubmits:
+  kubernetes/cloud-provider-vsphere:
+  - name: pull-cloud-provider-vsphere-test
+    agent: kubernetes
+    context: pull-cloud-provider-vsphere-test
+    branches:
+    - master
+    always_run: true
+    rerun_command: "/test pull-cloud-provider-vsphere-test"
+    trigger: "(?m)^/test( all| pull-cloud-provider-vsphere-test),?(\\s+|$)"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180716-9145034c9-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=execute"
+        - "--timeout=360"
```

Comment:
this should be `--timeout=6` if you want 6 minutes
https://github.com/kubernetes/test-infra/blob/fbef04d9c6af3e9c79c42da044d1e55e2a18c225/jenkins/bootstrap.py#L1138

Reply:
got it, thanks for point out, i misunderstood it when i saw another job set timeout at 60.
then let me set timeout to 30 minutes to give some room for new tests.

Reply:
yeah, lots of the tests actually take very long, and a lot more of them copy / pasted values from others. 🤷‍♂️ 

SGTM 👍